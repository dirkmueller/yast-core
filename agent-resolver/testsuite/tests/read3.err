[YCP] tests/read3.ycp:2 .resolver ["search", "nameserver", "domain", "sortlist", "options"]
[YCP] tests/read3.ycp:3 .resolver.modified dhcpcd
[YCP] tests/read3.ycp:4 .resolver.backup /etc/resolv.conf.saved.by.dhcpcd
[YCP] tests/read3.ycp:5 .resolver.process dhcpcd
[YCP] tests/read3.ycp:6 .resolver.process_id 1038
[YCP] tests/read3.ycp:7 .resolver.script /sbin/modify_resolvconf
[YCP] tests/read3.ycp:8 .resolver.saveto 
[YCP] tests/read3.ycp:9 .resolver.info This is a temporary resolv.conf created by service dhcpcd.
#               The previous file has been saved and will be restored later.
#               
#               If you don't like your resolv.conf to be changed, you
#               can set MODIFY_{RESOLV,NAMED}_CONF_DYNAMICALLY=no. This
#               variables are placed in /etc/sysconfig/network/config.
#               
#               You can also configure service dhcpcd not to modify it.
#               
#               If you don't like dhcpcd to change your nameserver
#               settings
#               then either set DHCLIENT_MODIFY_RESOLV_CONF=no
#               in /etc/sysconfig/network/dhcp, or
#               set MODIFY_RESOLV_CONF_DYNAMICALLY=no in
#               /etc/sysconfig/network/config or (manually) use dhcpcd
#               with -R.  If you only want to keep your searchlist, set
#               DHCLIENT_KEEP_SEARCHLIST=yes in /etc/sysconfig/network/dhcp or
#               (manually) use the -K option.
#
[YCP] tests/read3.ycp:10 .resolver.search nil
[YCP] tests/read3.ycp:11 .resolver.domain suse.cz
