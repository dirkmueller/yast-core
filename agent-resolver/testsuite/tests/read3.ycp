{
    y2warning(".resolver %1",SCR::Dir(.));
    y2warning(".resolver.modified %1",SCR::Read(.modified));
    y2warning(".resolver.backup %1",SCR::Read(.backup));
    y2warning(".resolver.process %1",SCR::Read(.process));
    y2warning(".resolver.process_id %1",SCR::Read(.process_id));
    y2warning(".resolver.script %1",SCR::Read(.script));
    y2warning(".resolver.saveto %1",SCR::Read(.saveto));
    y2warning(".resolver.info %1",SCR::Read(.info));
    y2warning(".resolver.search %1",SCR::Read(.search));
    y2warning(".resolver.domain %1",SCR::Read(.domain));
}
