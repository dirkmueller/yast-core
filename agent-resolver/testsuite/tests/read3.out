(nil)
### BEGIN INFO
#
# Modified_by:  dhcpcd
# Backup:       /etc/resolv.conf.saved.by.dhcpcd
# Process:      dhcpcd
# Process_id:   1038
# Script:       /sbin/modify_resolvconf
# Saveto:       
# Info:         This is a temporary resolv.conf created by service dhcpcd.
#               The previous file has been saved and will be restored later.
#               
#               If you don't like your resolv.conf to be changed, you
#               can set MODIFY_{RESOLV,NAMED}_CONF_DYNAMICALLY=no. This
#               variables are placed in /etc/sysconfig/network/config.
#               
#               You can also configure service dhcpcd not to modify it.
#               
#               If you don't like dhcpcd to change your nameserver
#               settings
#               then either set DHCLIENT_MODIFY_RESOLV_CONF=no
#               in /etc/sysconfig/network/dhcp, or
#               set MODIFY_RESOLV_CONF_DYNAMICALLY=no in
#               /etc/sysconfig/network/config or (manually) use dhcpcd
#               with -R.  If you only want to keep your searchlist, set
#               DHCLIENT_KEEP_SEARCHLIST=yes in /etc/sysconfig/network/dhcp or
#               (manually) use the -K option.
#
### END INFO
domain suse.cz
nameserver 10.20.0.2
nameserver 10.20.16.1
